app/assets/javascripts/discourse/app/lib/intercept-click.js

```diff
@@ -18,29 +18,33 @@ export function wantsNewWindow(e) {
   This jQuery code intercepts clicks on those links and routes them properly.
 **/
 export default function interceptClick(e) {
-  if (wantsNewWindow(e)) {
+  const target = e.target.closest("a");
+
+  if (!target) {
+    return;
+  }
+
+  if (wantsNewWindow(e, target) || target.target === "_blank") {
     return;
   }
 
-  const currentTarget = e.currentTarget;
-  const href = currentTarget.getAttribute("href");
-  const linkTarget = currentTarget.getAttribute("target");
-  const targettingOtherFrame = linkTarget && linkTarget !== "_self";
+  const href = target.getAttribute("href");
+  const linkTarget = target.getAttribute("target");
+  const targetingOtherFrame = linkTarget && linkTarget !== "_self";
 
   if (
     !href ||
     href.startsWith("#") ||
-    targettingOtherFrame ||
-    currentTarget.dataset.emberAction ||
-    currentTarget.dataset.autoRoute ||
-    currentTarget.dataset.shareUrl ||
-    currentTarget.classList.contains("widget-link") ||
-    currentTarget.classList.contains("raw-link") ||
-    currentTarget.classList.contains("mention") ||
-    (!currentTarget.classList.contains("d-link") &&
-      !currentTarget.dataset.userCard &&
-      currentTarget.classList.contains("ember-view")) ||
-    currentTarget.classList.contains("lightbox") ||
+    targetingOtherFrame ||
+    target.dataset.autoRoute ||
+    target.dataset.shareUrl ||
+    target.classList.contains("widget-link") ||
+    target.classList.contains("raw-link") ||
+    target.classList.contains("mention") ||
+    (!target.classList.contains("d-link") &&
+      !target.dataset.userCard &&
+      target.classList.contains("ember-view")) ||
+    target.classList.contains("lightbox") ||
```

Comment:
Noting the change from `currentTarget` to `target`.. 

`currentTarget` was `e.currentTarget`, but `target` is `e.target.closest("a").. is it the same thing?

Reply:
I hope so! 😜 

We were using the `selector` arg to jQ's `on()` which meant that it was doing a filtering, similar to `closest("a")` we have now.